requirements.txt

```diff
@@ -1,9 +1,10 @@
 click >=7
 ocrd >= 2.35.0
-pyyaml
+pyaml
 requests
 pipgrip
 nextflow
 tomli
 fastapi
 uvicorn[standard]
+pytest
```

Comment:
This should not be in the `requirements.txt` but in a `requirements-test.txt` or similar. You don't need it to run the software, just for testing/development.